src/main/java/in/koreatech/koin/admin/shop/dto/AdminShopsReviewsResponse.java

```diff
@@ -0,0 +1,142 @@
+import in.koreatech.koin.domain.shop.model.*;
+import in.koreatech.koin.global.model.Criteria;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.springframework.data.domain.Page;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+
+@JsonNaming(value = PropertyNamingStrategies.SnakeCaseStrategy.class)
+public record AdminShopsReviewsResponse(
+
+    @Schema(description = "총 상점의 수", example = "57", requiredMode = REQUIRED)
+    Long totalCount,
+
+    @Schema(description = "현재 페이지에서 조회된 수", example = "10", requiredMode = REQUIRED)
+    Integer currentCount,
+
+    @Schema(description = "현재 페이지", example = "2", requiredMode = REQUIRED)
+    Integer currentPage,
+
+    @Schema(description = "리뷰 리스트", requiredMode = REQUIRED)
+    List<InnerReviewResponse> reviews
+) {
+    public static AdminShopsReviewsResponse of(Page<ShopReview> reviews, Criteria criteria) {
+        return new AdminShopsReviewsResponse(
+            reviews.getTotalElements(),
+            reviews.getNumberOfElements(),
+            criteria.getPage(),
+            reviews.stream().map(InnerReviewResponse::from).toList()
+        );
+    }
+    public record InnerReviewResponse(
+        @Schema(example = "1", description = "리뷰 ID", requiredMode = REQUIRED)
+        int reviewId,
+
+        @Schema(example = "4", description = "별점", requiredMode = REQUIRED)
+        int rating,
+
+        @Schema(example = "익명23432423", description = "닉네임", requiredMode = REQUIRED)
+        String nickName,
+
+        @Schema(example = "맛있어요~", description = "내용", requiredMode = REQUIRED)
+        String content,
+
+        @Schema(example = """
+                [ "https://static.koreatech.in/example.png" ]
+                """, description = "이미지 URL 리스트", requiredMode = REQUIRED)
+        List<String> imageUrls,
+
+        @Schema(example = """
+                [ "피자" ]
+                """, description = "메뉴 이름 리스트", requiredMode = REQUIRED)
+        List<String> menuNames,
+
+        @Schema(example = "false", description = "수정된 적 있는 리뷰인지", requiredMode = REQUIRED)
+        boolean isModified,
+
+        @Schema(example = "false", description = "핸들링 되지 않은 신고가 있는 리뷰인지", requiredMode = REQUIRED)
+        boolean isHaveUnhandledReport,
+
+        @JsonFormat(pattern = "yyyy-MM-dd")
+        @Schema(example = "2024-03-01", description = "리뷰 작성일", requiredMode = REQUIRED)
+        LocalDateTime createdAt,
+
+        @Schema(description = "리뷰에 대한 신고 리스트")
+        List<InnerReportResponse> reports,
+
+        @Schema(description = "상점 정보", requiredMode = REQUIRED)
+        InnerShopResposne shop
+    ) {
+
+        public static InnerReviewResponse from(ShopReview review) {
+            String nickName = review.getReviewer().getUser().getNickname();
+            if (nickName == null) {
+                nickName = review.getReviewer().getAnonymousNickname();
+            }
```

Comment:
따로 변수 지정하지말고, 삼항연산자를 쓰는것은 어떤것같으신가요? (코드 취향 질문)

Reply:
@BaeJinho4028 
제가 개인적으로 삼항연잔사를 별로 안좋아하기는 하는데 java에는 optional이라는 좋은 친구가 있다는것을 이제 알았네요
수정했습니다
```suggestion
String nickName = Optional.ofNullable(review.getReviewer().getUser().getNickname())
                                      .orElse(review.getReviewer().getAnonymousNickname());
```